package.json

```diff
@@ -58,8 +58,8 @@
     "@ethersproject/abstract-signer": "^5.6.2",
     "@ethersproject/constants": "^5.6.1",
     "@nomiclabs/hardhat-etherscan": "^2.1.6",
-    "@openzeppelin/contracts": "^4.7.3",
-    "@openzeppelin/contracts-upgradeable": "^4.7.3",
+    "@openzeppelin/contracts": "^4.8.0",
+    "@openzeppelin/contracts-upgradeable": "^4.8.0",
```

Comment:
We can use the https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.8.1 which replaces the call to staticcall

My this PR will update it later.
https://github.com/bcnmy/scw-contracts/pull/40/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519